etc/schema.json

```diff
@@ -7083,6 +7115,7 @@
                 },
                 "defrag": {
                     "type": "object",
+                    "description": "Statistics for the IP fragmentation engine",
```

Comment:
I don't know if it's ok, but we have a defrag engine. It does defragmentation of IP fragments FWIU